### PR TITLE
Dr 22 10 sinai et002

### PR DESCRIPTION
**Pull request by thea-m**

### CODE REVIEW

**Comment by thea-m** [comment]
I don't understand the error message, the file is valid

File: SinaiStCatherine/SinaiEt002.xml [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+                 </origin>
+               </history>
+               <additional>
+                  <adminInfo>
+                     <recordHist>
+                       <source>
+                         <listBibl type="catalogue">
+                           <bibl>
+                             <ptr target="bm:Kamil1957Sinai"/>
+                             <citedRange unit="page">85-86</citedRange>
```
Comment (DenisNosnitsin1970) [inline SinaiStCatherine/SinaiEt002.xml:242]: Indicate number?

**Review by DenisNosnitsin1970 — APPROVED** [review]

**Comment by eu-genia** [comment]
> I don't understand the error message, the file is valid

it has nothing to do with validation, i have already asked @duncdrum to have a look. we now will be getting the error on every PR until this is fixed in the CI

**Review by eu-genia — APPROVED** [review]

**Comment by duncdrum** [comment]
ci is working again, please rebase this PR to get proper validation results.

**Pull request merged**